api/src/lib/connector.js

```diff
@@ -117,26 +115,24 @@ module.exports = class Conncetor {
     const hostInfo = yield this.getPeerInfo(peer)
 
     try {
+      let options = {
+        name: peer.hostname,
+        secret: this.config.data.getIn(['connector', 'ed25519_secret_key']),
+        peerPublicKey: hostInfo.publicKey,
+        prefix: hostInfo.ledgerName,
+        rpcUri: hostInfo.rpcUri,
+        maxBalance: '' + peer.limit,
+        currencyScale: peer.currencyScale || 9,
```

Comment:
The default value here and on line 93 should also go into a constant. I guess it would make sense to have a single constant `DEFAULT_CURRENCY_SCALE` ILP Kit wide.